features/FxA-54-push-on-status-change/README.md

```diff
@@ -0,0 +1,84 @@
+# Push Notifications for Account Status Changes
+
+https://mozilla.aha.io/features/FXA-54
+
+## Stories
+
+As a user signed in to an FxA connected device,
+I want my device to update its UI in a timely manner
```

Comment:
@rfk Should we add detail to this or leave it? 

"I want my device to update its UI in a timely manner..." -> "I want my device to update its FxA login state when I reset my password or delete my account from another device"